GeoLib/Polyline.cpp

```diff
@@ -94,7 +94,9 @@ void Polyline::insertPoint(std::size_t pos, std::size_t pnt_id)
 		}
 	}
 
-	std::vector<std::size_t>::iterator it(_ply_pnt_ids.begin() + pos);
+	std::vector<std::size_t>::iterator it(
+	    _ply_pnt_ids.begin() +
+	    static_cast<std::vector<std::size_t>::difference_type>(pos));
```

Comment:
Minor thing: I think the code would be much more readable if you you'd provide a `const pos_signed` once in this method and would not cast many times. Or does `pos` change? At least it's not const.